app/templates/src/test/javascript/spec/app/account/activate/_activate.controller.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-describe('Controllers Tests ', function() {
+describe('Controller Tests', function() {
```

Comment:
I'd recommend going a step further and naming these something like 'Activate Controller Tests'.